### ISSUE TITLE
loading and filtering overlaps failed 

### ISSUE THREAD

**Issue by reel-cmd** [marbl/canu#1645]
Hello, 

I am trying to assemble nanopore metagenomic data from a human stool sample on our HPCC (SLURM). 

I have tried increasing the memory but can’t get past the “LOADING AND FILTERING OVERLAPS” step.  

Version: Canu 1.8
```
Command: 
canu \
useGrid=true \
gridEngineArrayMaxJobs=10 \
gridOptions="--time=6-00:00:00 --mem=500G --partition=preempt --reservation=galaxyres " \
corOutCoverage=10000 \
corMhapSensitivity=high \
corMinCoverage=0 \
redMemory=200 \
oeaMemory=200 \
batMemory=300 \
-p prefix \
-d directory \
genomeSize=5m \ (but have also tried others like 1g) 
-nanopore-raw file.fastq

Error: 
ABORT:   ==> LOADING AND FILTERING OVERLAPS.
ABORT:   
ABORT:   ReadInfo()-- Using 1713046 reads, no minimum read length used.
ABORT:   
ABORT:   OverlapCache()-- limited to 204800MB memory (user supplied).
ABORT:   
ABORT:   OverlapCache()--      13MB for read data.
ABORT:   OverlapCache()--      65MB for best edges.
ABORT:   OverlapCache()--     169MB for tigs.
ABORT:   OverlapCache()--      45MB for tigs - read layouts.
ABORT:   OverlapCache()--      65MB for tigs - error profiles.
ABORT:   OverlapCache()--   51200MB for tigs - error profile overlaps.
ABORT:   OverlapCache()--       0MB for other processes.
ABORT:   OverlapCache()-- ---------
ABORT:   OverlapCache()--   51592MB for data structures (sum of above).
ABORT:   OverlapCache()-- ---------
ABORT:   OverlapCache()--      32MB for overlap store structure.
ABORT:   OverlapCache()--  153175MB for overlap data.
ABORT:   OverlapCache()-- ---------
ABORT:   OverlapCache()--  204800MB allowed.
ABORT:   OverlapCache()--
ABORT:   OverlapCache()-- Retain at least 6800 overlaps/read, based on 3400.39x coverage.
ABORT:   OverlapCache()-- Initial guess at 5860 overlaps/read.
ABORT:   OverlapCache()--
ABORT:   OverlapCache()-- Not enough memory to load the minimum number of overlaps; increase -M.
```
Let me know if I can provide any more details. Thank you for your insight!

**Comment by skoren** [comment]
You need to increase either batMemory or genome size, but you also have to remove the 4-unitigging folder first. Have you done both?

**Comment by skoren** [comment]
Idle, similar to #1650 and previous errors due to high coverage/small genome size which is addressed by automatic downsampling in v2.0
